templates/repo/editor/edit.tmpl

```diff
@@ -15,7 +15,8 @@
 						{{range $i, $v := .TreeNames}}
 							<div class="divider"> / </div>
 							{{if eq $i $l}}
-								<input id="file-name" value="{{$v}}" placeholder="{{$.i18n.Tr "repo.name_your_file"}}" required autofocus> 
+								<input id="file-name" value="{{$v}}" placeholder="{{$.i18n.Tr "repo.name_your_file"}}" required autofocus
+									data-ec-url="{{$.AppSubUrl}}/api/v1/repos{{$.RepoLink}}/editorconfig/{{$.Repository.DefaultBranch}}/">
```

Comment:
`{{$.RepoLink}}` is a link, not just owner and repository name, I don't think it works..